apps/builder/app/builder/features/topbar/domain-checkbox.tsx

```diff
@@ -0,0 +1,82 @@
+import { useStore } from "@nanostores/react";
+import {
+  Flex,
+  Tooltip,
+  Text,
+  theme,
+  Link,
+  buttonStyle,
+  Checkbox,
+} from "@webstudio-is/design-system";
+import { $userPlanFeatures } from "~/builder/shared/nano-states";
+import { $project } from "~/shared/nano-states";
+
+export const domainToPublishName = "domainToPublish[]";
+
+interface DomainCheckboxProps {
+  defaultChecked?: boolean;
+  domain: string;
+  buildId: string | undefined;
+  disabled?: boolean;
+}
+
+const DomainCheckbox = (props: DomainCheckboxProps) => {
+  const hasProPlan = useStore($userPlanFeatures).hasProPlan;
+  const project = useStore($project);
+
+  if (project === undefined) {
+    return;
+  }
+
+  const tooltipContentForFreeUsers = hasProPlan ? undefined : (
+    <Flex direction="column" gap="2" css={{ maxWidth: theme.spacing[28] }}>
+      <Text variant="titles">Publish to Staging</Text>
+      <Text>
+        <Flex direction="column">
```

Comment:
why flex with all line breaks?

Reply:
copypasted from the same tooltip, haven’t spent even second in this code